src/lib/view.ts

```diff
@@ -67,19 +70,14 @@ export const findChildrenAttrsByType = <T extends Attrs>(
   ) as T[]
 }
 
-export const updateNode = (
-  view: ManuscriptEditorView,
-  node: ManuscriptNode
-) => {
-  updateNodeAttrs(view, node.type, node.attrs)
-}
-
 export const updateNodeAttrs = (
   view: ManuscriptEditorView,
   type: ManuscriptNodeType,
-  attrs: Attrs
+  attrs: Attrs,
+  prevAttrs?: Attrs
 ) => {
-  const copy = { ...attrs }
+  const copy = prevAttrs ? sanitizeAttrsChange(attrs, prevAttrs) : { ...attrs }
```

Comment:
Certainly! Thanks for catching this. I've updated also [contributors.ts](https://github.com/Atypon-OpenSource/manuscripts-body-editor/pull/232/files#diff-b6b9376427a5d7851234906cef9b7a795bca7382cd6b55178002abae119e8703) so it can take advantage of this search for a child instead of performing its own.